### DISCUSSION TITLE
Thoughts on RIST? Any plans to add RIST support in the future?

### DISCUSSION THREAD

**Discussion by BuildMineSurvive** [bluenviron/mediamtx#2951] [General]
I think RIST support would be pretty cool. I've had instances where it worked better than SRT, but I'm not sure why. I'm pretty technical, but not digging into the weeds of transport protocol libraries technical. 

Has anyone else here used RIST? 

@aler9 I love this project! Starting to integrate it into my workflow for sending video feeds across the internet.  Do you have any plans to add RIST? Or do you consider it redundant to SRT, and not as well adopted?

**Comment by aler9** [comment]
duplicate of #677
